examples/network-policy/webhook-network-policy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: longhorn-webhook
+  namespace: longhorn-system
+spec:
+  podSelector:
+    matchLabels:
+      app: longhorn-webhook
+  policyTypes:
+  - Ingress
+  ingress:
+  - ports:
+    - protocol: TCP
```

Comment:
Does this work w/o UCP port allowed? for the case Mohamed found.

Reply:
It's the UDP port `8472` which is the Canal/Flannel VXLAN overlay networking
https://rancher.com/docs/rancher/v2.5/en/installation/requirements/ports/#commonly-used-ports

I found that each host open the same UDP port with the command:
```
ss -ulpn
```